politeiawww/config.go

```diff
@@ -708,36 +751,39 @@ func loadConfig() (*config.Config, []string, error) {
 				"and dbkey: %v", err)
 		}
 
-		// Validate user database encryption keys
-		cfg.EncryptionKey = util.CleanAndExpandPath(cfg.EncryptionKey)
```

Comment:
This introduced a bug.

`new cockroachdb db: load encryption key ~/.politeiawww/sbox.key: open ~/.politeiawww/sbox.key: no such file or directory`

You need to clean the `cfg.EncryptionKey` and `cfg.OldEncryptionKey` paths in both the cockroach and mysql code paths.

Reply:
oh, looks like i need to clean the paths before passing instead of cleaning them inside `validateEncryptionKey`.